crx_template/wyrmhole.js

```diff
@@ -254,23 +205,4 @@
             });
         };
     }
-
```

Comment:
The lines from here to the end are intended to make it possible to have multiple extensions installed and not conflict with each other; a similar solution is needed in your wyrmhole_content.js script